docs/tutorials/quickstart.md

```diff
@@ -1,8 +1,194 @@
+---
+jupytext:
+  formats: md:myst
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.15.2
+kernelspec:
+  display_name: Python 3
+  language: python
+  name: python3
+---
+
 # Quickstart
 
-```{note}
-This is a placeholder for a section in the new {ref}`jax-tutorials`.
+**JAX is a library for array-oriented numerical computation (*a la* [NumPy](https://numpy.org/)), with automatic differentiation and JIT compilation to enable high-performance machine learning research**.
+
+This document gives a quick overview of essential JAX features:
+
+* JAX provides a unified NumPy-like interface to computations that run on CPU, GPU, or TPU, in local or distributed settings.
+* JAX features built-in Just-In-Time (JIT) compilation via [Open XLA](https://github.com/openxla), an open-source machine learning compiler ecosystem.
+* JAX programs support efficient evaluation of gradients via its automatic differentiation transformations.
```

Comment:
```suggestion
* JAX functions support efficient evaluation of gradients via its automatic differentiation transformations.
```

@mattjj may have proposed a similar change elsewhere in the doc (JAX programs -> JAX functions)